python-sdk/example_dags/example_datasets.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime
+
+from airflow import DAG
+
+from astro import sql as aql
+from astro.files import File
+from astro.sql.table import Table
+
+input_file = File(path="gs://dag-authoring/movies/imdb_v2.csv")
```

Comment:
Should we add this dag as part of integration tests too?

thoughts @pankajastro @utkarsharma2 ?

Reply:
In my opinion yes, if we don't have any other DAG testing datasets compatibility.

Reply:
Will address this in https://github.com/astronomer/astro-sdk/issues/816